app/lib/dartdoc/models.dart

```diff
@@ -16,23 +16,58 @@ import 'storage_path.dart' as storage_path;
 
 part 'models.g.dart';
 
+/// Describes the details of a dartdoc-generated content.
 @JsonSerializable()
 class DartdocEntry {
+  /// Unique identifier of the entry, generated via a random UUID.
   final String uuid;
+
+  /// The package name.
```

Comment:
not needed

Reply:
I kind of like the consistent spacing of the fields (e.g. one empty line + comment + field), but sure, removing it.